modules/directives/src/main/scala/scala/build/preprocessing/directives/Sources.scala

```diff
@@ -23,7 +23,8 @@ import scala.util.Try
 final case class Sources(
   @DirectiveName("file")
     files: DirectiveValueParser.WithScopePath[List[Positioned[String]]] =
-      DirectiveValueParser.WithScopePath.empty(Nil)
+      DirectiveValueParser.WithScopePath.empty(Nil),
+   exclude: List[Positioned[String]] = Nil
```

Comment:
It seems that the reference documentation is not updated - it seems that the mechanism to detect changes and/or generate the reference is broken.